### DISCUSSION TITLE
启动报错

### DISCUSSION THREAD

**Discussion by Lucien-Harvey** [TriDiamond/hexo-theme-obsidian#174] [Q&A]
![obsidian](https://user-images.githubusercontent.com/114826497/195039364-7c984508-d366-4619-a474-96b97332c5b4.jpg)
这是怎么回事1

**Comment by aisuandebowen** [comment]
根目录创建 _config.obsidian.yml
配置参考themes/obsidian/_config_template_cn.yml
